Examples/ObjectiveC/FeatureSelectionExample.m

```diff
@@ -85,13 +83,10 @@ - (void)changeOpacityBasedOn:(NSString*)name {
     
     // Check if a state was selected, then change the opacity of the states that were not selected.
     if (name.length > 0) {
-        layer.fillOpacity = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeCategorical
-                sourceStops:@{ name: [MGLStyleValue valueWithRawValue:@1] }
-                attributeName:@"name"
-                options:@{ MGLStyleFunctionOptionDefaultValue: [MGLStyleValue valueWithRawValue:@0] }];
+        layer.fillOpacity = [NSExpression expressionWithFormat:@"TERNARY(FUNCTION(name, 'stringValue') = %@, 1, 0)", name];
```

Comment:
Instead of calling the `stringValue` custom function, use `CAST(name, 'NSString')`.